core/src/test/java/org/infinispan/container/SimpleDataContainerTest.java

```diff
@@ -59,38 +66,39 @@ public void testExpiredData() throws InterruptedException {
       InternalCacheEntry entry = dc.get("k");
       assertNotNull(entry);
       assertEquals(transienttype(), entry.getClass());
-      assertTrue(entry.getLastUsed() <= System.currentTimeMillis());
+      assertTrue(entry.getLastUsed() <= timeService.wallClockTime());
       long entryLastUsed = entry.getLastUsed();
-      Thread.sleep(100);
+      timeService.advance(100);
       entry = dc.get("k");
       assertTrue(entry.getLastUsed() > entryLastUsed);
       dc.put("k", "v", new EmbeddedMetadata.Builder().maxIdle(0, TimeUnit.MINUTES).build());
```

Comment:
I guess to make sure the metadata is overridden properly. I can't say otherwise.  Seems safe to leave.